test/new-e2e/tests/process/k8s_test.go

```diff
@@ -213,6 +217,11 @@ func (s *K8sSuite) TestProcessCheckInCoreAgentWithNPM() {
 		status := k8sAgentStatus(t, s.Env().KubernetesCluster)
 		assert.ElementsMatch(t, []string{"process", "rtprocess"}, status.ProcessComponentStatus.Expvars.Map.EnabledChecks)
 		assert.ElementsMatch(t, []string{"connections"}, status.ProcessAgentStatus.Expvars.Map.EnabledChecks)
+
+		if t.Failed() {
+			t.LogF("status: %+v\n", status)
```

Comment:
Just curious, would `t.logf` work in this case? From documentation [here](https://pkg.go.dev/testing#T.Logf) it seems like "the text will be printed only if the test fails" which is exactly what the if statement and logF replicate.

Reply:
Its the same function, I just had a typo 😆 . Though, the fail check is there because the tests are run with the `-v` flag so the log would always show otherwise.